documentation/modules/ROOT/pages/connectors/db2.adoc

```diff
@@ -217,6 +306,34 @@ If the default topic name do not meet your requirements, you can configure custo
 To configure custom topic names, you specify regular expressions in the logical topic routing SMT.
 For more information about using the logical topic routing SMT to customize topic naming, see {link-prefix}:{link-topic-routing}#topic-routing[Topic routing].
 
+// Type: concept
+// ModuleID: how-debezium-db2-connectors-handle-database-schema-changes
+// Title: How {prodname} Db2 connectors handle database schema changes
+[[db2-schema-history-topic]]
+=== Schema history topic
+
+When a database client queries a database, the client uses the database’s current schema.
+However, the database schema can be changed at any time, which means that the connector must be able to identify what the schema was at the time each insert, update, or delete operation was recorded.
+Also, a connector cannot necessarily apply the current schema to every event.
+If an event is relatively old, it's possible that it was recorded before the current schema was applied.
+
+To ensure correct processing of changes that occur after a schema change, Db2 includes in the binlog not only the row-level changes that affect the data, but also the DDL statements that are applied to the database.
+As the connector reads the binlog and comes across these DDL statements, it parses them and updates an in-memory representation of each table’s schema.
+The connector uses this schema representation to identify the structure of the tables at the time of each insert, update, or delete operation and to produce the appropriate change event.
+In a separate database schema history Kafka topic, the connector records all DDL statements along with the position in the binlog where each DDL statement appeared.
+
+When the connector restarts after either a crash or a graceful stop, it starts reading the binlog from a specific position, that is, from a specific point in time.
+The connector rebuilds the table structures that existed at this point in time by reading the database schema history Kafka topic and parsing all DDL statements up to the point in the binlog where the connector is starting.
+
+This database schema history topic is for connector use only.
+Optionally, the connector can also xref:about-the-debezium-db2-connector-schema-change-topic[emit schema change events to a different topic that is intended for consumer applications].
+
+When the Db2 connector captures changes in a table to which a schema change tool such as `gh-ost` or `pt-online-schema-change` is applied, there are helper tables created during the migration process.
+The connector needs to be configured to capture change to these helper tables.
+If consumers do not need the records generated for helper tables, then a single message transform can be applied to filter them out.
```

Comment:
This content appears in the MySQL connector schema history topic. Does it also apply to Db2, Oracle, and SQL Server?

Reply:
These are only MySQL specific.

Reply:
> These are only MySQL specific.

Ah, ok. Is the use of such tools in general also specific to MySQL? That is, for another DB, might a user apply a different schema change tool that would also generate files that the connector should capture?
Or should I remove these 3 lines from the doc for the other connectors?

Reply:
I'm not aware of any, so I would recommend removing these from the other connectors.